frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp

```diff
@@ -209,6 +211,12 @@
                 {title: "Moderators", render: {display: renderModeratorsListExpandable}},
                 {title: "State"},
                 {title: "Actions"}
+            ],
+            columnDefs: [
+                {
+                    width: "90px", "targets": [0],
```

Comment:
Why does datables needs the width here? does it ignoire the width stated in the html above?

Reply:
just a test, removed code.